assets/next-thursday.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+function nextNthWeekday(date, numDay, occurance) {
+	// weekday starts at 0
+	// month starts at 0
+	// date... starts at 1
+	numDay = parseInt(numDay, 10);
+	occurance = parseInt(occurance, 10);
+	if (!(occurance >= 1 && occurance <= 5)) {
+		throw new Error('occurance must be >= 1 and <= 5');
+	}
+	if (!(numDay >= 0 && numDay <= 7)) {
+		throw new Error('weekday must be >= 0 and <= 7');
+	}
+	numDay = numDay % 7;
+
+	var yyyy = date.getFullYear();
+	var m = date.getMonth();
+	var d = 1; // date.getDate();
+	var di = 0;
```

Comment:
i think you need to run prettier on this file again

Reply:
@solderjs: Is this actually an issue?

Reply:
I ran it. Again.

I'd highly recommend that we pick a few linters to standardize on. I vote prettier because it's what everyone uses and obstinate old me gave in and started using it.

Reply:
I'll resolve this, then. @TomPridham, not sure what you were thinking here, but it doesn't look like re-running `prettier` yielded any differences.